src/Symfony/Component/DependencyInjection/ContainerBuilder.php

```diff
@@ -1440,6 +1440,23 @@ public function log(CompilerPassInterface $pass, $message)
         $this->getCompiler()->log($pass, $message);
     }
 
+    /**
+     * @param $prefix
+     * @param string $prefix
+     *
+     * @return string
+     */
+    public function generateServiceName($prefix)
```

Comment:
should be moved as a private method to the ServicesConfigurator as there is no point in exposing such a method

Reply:
I think it's possible to implement the same feature for other configurations. `ServicesConfigurator` is for fluent-format only, isn't it?